### PR TITLE
Add project ID argument for CLI create task

### PR DESCRIPTION
**Pull request by adamhrv**
### Motivation and context

The new project feature is helpful, but it is currently not compatible with the CLI create task script. This PR adds the `--project_id` option for the create task and sets the labels to an empty list, which defaults to the project's labels.

### How has this been tested?

Add the `--project_id [id]` argument. For example:
```
docker exec -it cvat bash -ic \
  "python3 utils/cli/cli.py \
  --auth ${CVAT_USERNAME}:${CVAT_PASSWORD} \
  create --project_id 1 TaskName local /path/to/file.jpg"
```

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: utils/cli/core/definition.py [hunk 1]
```diff
@@ -162,6 +162,13 @@ def argparse(s):
     help='''number of seconds to wait until checking
             if data compression finished (necessary before uploading annotations)'''
 )
+task_create_parser.add_argument(
+    '--project_id',
```
Comment (nmanovic) [inline utils/cli/core/definition.py:166]: Let's name it as `--project`.

**Comment by jahaniam** [comment]
@ActiveChooN It would be if we could move a task to a project i.e. if the task project id is None, we could assign a project ID to them through API. This way we could move a task that is not in any project to a project (for people who are upgrading their cvat to use this feature)

File: utils/cli/core/definition.py (hunk 1 above)
Comment (adamhrv) [reply utils/cli/core/definition.py:166]: the more verbose `project_id` helps disambiguate between project ID and project name

**Comment by bsekachev** [comment]
@adamhrv
Hi, something wrong with this PR now. It shows changes that actually were done in other patches.

**Comment by adamhrv** [comment]
sorry didn't realize this would mess up the PR from 2 days ago. I merged the latest commits from this `develop` branch to my `develop` branch fork thinking it would sync up, but that obviously didn't work as intended. is there way to untangle this? otherwise I'll close and resubmit a fresh PR. it's a rather small change.

**Comment by bsekachev** [comment]
> sorry didn't realize this would mess up the PR from 2 days ago. I merged the latest commits from this `develop` branch to my `develop` branch fork thinking it would sync up, but that obviously didn't work as intended. is there way to untangle this? otherwise I'll close and resubmit a fresh PR. it's a rather small change.

It would be better to try fixing this PR.
Something like: 
```sh
git checkout openvinotoolkit/develop
git pull openvinotoolkit develop
git checkout develop
git reset --hard 61842627512  # hash of your latest commit (check if it correct)
git merge openvinotoolkit/develop
git push origin develop --force # be sure all changes are saved before force pushing
```

**Comment by nmanovic** [comment]
@adamhrv , I will close the PR. Please reopen/re-create it if you are ready to resolve all problems with merge.

**Pull request closed without merging**